mod-kb-ebsco/mod-kb-ebsco-java.postman_collection.json

```diff
@@ -7703,6 +7703,119 @@
 					}
 				}
 			]
+		},
+		{
+			"name": "status",
```

Comment:
@ivan-brovenko -- the status folder should appear before the tear-down configuration folder in the postman collection -- reason being is that configuration may be cleared due to this. ![screen shot 2018-11-27 at 4 46 15 pm](https://user-images.githubusercontent.com/19415226/49113738-4bf81100-f264-11e8-969c-41ad9f34311d.png)

Also -- I think we will want both positive and negative test for this endpoint eventually. This PR will can address the postiives. Added a Jira ticket https://issues.folio.org/browse/MODKBEKBJ-103 for cache invalidate and a ticket to address negative tests https://issues.folio.org/browse/MODKBEKBJ-104